keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -229,20 +306,37 @@ https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
+If strict decoding fails, proxy fall back to a lenient decoder, ensuring  that
+rollouts and rollbacks remain unaffected in case of a malformed configuration.
+
+Running workloads are not affected.
+
 ###### What specific metrics should inform a rollback?
 
+N/A
+
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+N/A
+
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
+No. 
```

Comment:
Won't we deprecate v1alpha1?

Reply:
No. The so-called-alpha component config version has been the _only_ config version for years, and for a while we were mistakenly recommending that people use the alpha config file rather than the non-alpha command-line arguments. (https://github.com/kubernetes/kubernetes/issues/118090).

So anyway, there are lots of `v1alpha` configs out there, and even lots of "things that generate `v1alpha` configs" (like kubeadm). So we expect to keep supporting that for a while, but just not improving it.

(There's some discussion above (https://github.com/kubernetes/enhancements/pull/4337#discussion_r1460863776) about whether we really need to guarantee full round-trip-ability between config file versions, or if k8s only expects that for API types, not config types.)

Reply:
(OTOH I do expect to deprecate and eventually remove `v1alpha2`)